releases/release-5.1.5.md

```diff
@@ -0,0 +1,161 @@
+    - Fix the metadata corruption issue when `Prepare Merge` is triggered after a new election is finished but the isolated peer is not informed [#3435](https://github.com/pingcap/tiflash/issues/3435) @[solotzg](https://github.com/solotzg)
+    (dup: release-5.3.2.md > Bug Fixes> TiFlash)- Fix a bug that an MPP query might fail due to random gRPC keepalive timeout [#4662](https://github.com/pingcap/tiflash/issues/4662)
+    (dup: release-5.4.1.md > Bug Fixes> TiFlash)- Fix the issue that the result of `IN` is incorrect in multi-value expressions [#4016](https://github.com/pingcap/tiflash/issues/4016)
+    (dup: release-5.4.1.md > Bug Fixes> TiFlash)- Fix a bug that MPP tasks might leak threads forever [#4238](https://github.com/pingcap/tiflash/issues/4238)
+    (dup: release-5.4.1.md > Bug Fixes> TiFlash)- Fix the issue that expired data is recycled slowly [#4146](https://github.com/pingcap/tiflash/issues/4146)
+    (dup: release-5.4.1.md > Bug Fixes> TiFlash)- Fix the overflow that occurs when casting `FLOAT` to `DECIMAL` [#3998](https://github.com/pingcap/tiflash/issues/3998)
+    (dup: release-5.3.4.md > Bug fixes> TiFlash)- Fix the issue that logical operators return wrong results when the argument type is UInt8 [#6127](https://github.com/pingcap/tiflash/issues/6127)
+    - None [#5849](https://github.com/pingcap/tiflash/issues/5849) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    (dup: release-5.3.2.md > Bug Fixes> TiFlash)- Fix the potential `index out of bounds` error if calling `json_length` with empty string [#2705](https://github.com/pingcap/tiflash/issues/2705)
+    (dup: release-5.3.2.md > Bug Fixes> TiFlash)- Fix wrong decimal comparison results in corner cases [#4512](https://github.com/pingcap/tiflash/issues/4512)
+    (dup: release-5.3.2.md > Bug Fixes> TiFlash)- Fix `TiFlash_schema_error` reported when `NOT NULL` columns are added [#4596](https://github.com/pingcap/tiflash/issues/4596)
+    - [#4091](https://github.com/pingcap/tiflash/issues/4091) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    - None [#5979](https://github.com/pingcap/tiflash/issues/5979) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    - None [#4091](https://github.com/pingcap/tiflash/issues/4091) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    - [#4091](https://github.com/pingcap/tiflash/issues/4091) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    - None [#5385](https://github.com/pingcap/tiflash/issues/5385) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    - None [#4405](https://github.com/pingcap/tiflash/issues/4405) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    (dup: release-5.3.4.md > Bug fixes> TiFlash)- Fix the issue that TiFlash bootstrap fails when `0.0` is used as the default value for integers, for example, `` `i` int(11) NOT NULL DEFAULT '0.0'`` [#3157](https://github.com/pingcap/tiflash/issues/3157)
+
++ Tools
+
```

Comment:
+ TiDB-Binlog
        - Fix the bug that Drainer can't send request correctly to Pump when setting `compressor=gzip` [#1152](https://github.com/pingcap/tidb-binlog/issues/1152) @[lichunzhu](https://github.com/lichunzhu)